doc/README.md

```diff
@@ -34,6 +34,45 @@ $ nix-build doc
 
 If the build succeeds, the manual will be in `./result/share/doc/nixpkgs/manual.html`.
 
+### Shell utilities
+
+In order to ease up your contribution workflow, consider loading the provided shell by using `nix-shell`.
+
+You can load it from the root of the nixpkgs tree with:
+
+```ShellSession
+$ cd /path/to/nixpkgs
+$ nix-shell doc
+```
+
+Alternatively, you can load it by first entering the doc folder:
+
+```ShellSession
+$ cd /path/to/nixpkgs/doc
+$ nix-shell
+```
```

Comment:
````suggestion
Load it from the Nixpkgs documentation directory with:

```ShellSession
$ cd /path/to/nixpkgs/doc
$ nix-shell
```
````

Let's not provide multiple (almost?) equivalent options, it's complicated enough as it is. If there's any reason to prefer to start at the repo root, use that of course. I picked the doc dir because with the direnv setup you simply skip the second command.